imports/plugins/core/collections/lib/registerSchema.js

```diff
@@ -0,0 +1,25 @@
+export const Schemas = {}; // populated with all Schemas
+
+/**
+ * Register a schema
+ * @param {String} name The name of the schema to register.
+ * @param {Array} schema Schema data.
+ *
+ * @returns {Empty} no return. Adds schema to Schemas object
+ */
+export function registerSchema(name, schema) {
+  if (!name || !schema) {
+    throw new Error("A name and schema object are required for registerSchema");
+  }
+
+  // store the component in the table
+  Schemas[name] = schema;
```

Comment:
Should we match the style of `registerComponent` and do `Schemas[name] = { name, schema }` ?